linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,62 @@
+name: nextcloud-client
+version: 2.2.4-git
+summary: nextcloud desktop client
+icon: ../client/theme/colored/owncloud-icon.svg
+description: |
+  The ownCloud Desktop Client is a tool to synchronize files from ownCloud
+  Server with your computer.
+
+grade: devel
+confinement: strict
```

Comment:
Ok, agree... Still not sure if devmode is better though as an interim step to the final confined (if that is ever wanted).